examples/digital_in/cont_wfm_acq_int_clk.py

```diff
@@ -0,0 +1,28 @@
+﻿"""Example for reading digital signal.
+
+This example demonstrates how to acquire a continuous digital
+waveform using the DAQ device's internal clock.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, LineGrouping, READ_ALL_AVAILABLE
+
+
+with nidaqmx.Task() as task:
+    task.di_channels.add_di_chan("Dev1/port0/line0:3", line_grouping=LineGrouping.CHAN_PER_LINE)
+    task.timing.cfg_samp_clk_timing(1000.0, sample_mode=AcquisitionType.CONTINUOUS)
+    task.start()
+    print("Acquiring samples continuously. Press Ctrl+C to stop.")
+
+    try:
+        total_read = 0
+        while True:
+            data = task.read(READ_ALL_AVAILABLE)
```

Comment:
Don't use READ_ALL_AVAILABLE for the continuous examples. For continuous mode, READ_ALL_AVAILABLE reads whatever is in the DAQmx circular buffer, which might be a small number of samples like 0 or 4.

Instead, specify a number of samples to read, like you did for https://github.com/ni/nidaqmx-python/blob/master/examples/analog_in/cont_voltage_acq_int_clk.py